linq4j/src/main/java/org/apache/calcite/linq4j/EnumerableDefaults.java

```diff
@@ -1054,7 +1054,31 @@ public void close() {
         resultSelector,
         comparer,
         generateNullsOnLeft,
-        generateNullsOnRight);
+        generateNullsOnRight, null);
```

Comment:
minor: to be consistent style-wise, `null` should be in a new line (same with `predicate` in line 1081).